staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -20,6 +20,9 @@ import (
 	"fmt"
 
 	"github.com/spf13/pflag"
+	"net/http"
+	"k8s.io/apiserver/pkg/server/healthz"
+	"k8s.io/kubernetes/cmd/kube-apiserver/app/preflight"
```

Comment:
Is the reverse allowed, that is adding a `k8s.io/apiserver/...` dependency to `k8s.io/kubernetes/cmd/kube-apiserver/app/preflight`? So I don't end up duplicating code